### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by sanketwagh147**
Modified 2 things

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Define the main function
+
+def main():
+    print("Welcome To Version 1")
```
Comment (sanketwagh1808) [inline main.py:4]: Add a yay function that prints yay

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # prdemo
-This is to learn about Pull request 
+# This is to learn about Pull request
```
Comment (sanketwagh1808) [inline README.md:2]: Add third line

File: main.py (hunk 1 above)
Comment (FireArmz) [reply main.py:4]: okay will add it soon

File: README.md (hunk 2 above)
Comment (FireArmz) [reply README.md:2]: what should I add ?

Comment (sanketwagh1808) [reply README.md:2]: Tomato Tomata

**Review by sanketwagh1808 — APPROVED** [review]

**Comment by sanketwagh1808** [comment]
Thank you

**Pull request merged**